src/UI/App.jsx

```diff
@@ -504,6 +512,7 @@ export default class App extends React.Component {
 		Logger.debug("Show chat, registering device");
 
 		this.setState(() => ({showChat: true}));
+		localStorage.setItem("messengerOpenState", "open");
```

Comment:
Ik wil dat overal waar `open` of `minimize` staat (in je nieuwe code) dat je constants gaat gebruiken zodat we niet perongeluk een typefout kunnen maken

Reply:
Ik heb deze toegevoegd als class properties, omdat constants binnen constructor niet aangeroepen kunnen worden van buitenaf.

Zie: https://github.com/parley-messaging/web-library/pull/39/files#diff-0abef088b3f6260f845818ba7eef7e55a43e70e212c423ebaa54066f4c94aba7R35

Reply:
Dit is prima voor nu
Resolved, fixed strings zitten nu in een var

Reply:
Ik heropen deze toch nog even, een mooiere aanpassing is 
```js
this.messengerState = {
  open: 'open',
  minimize: 'minimize'
}
```

Dan kan je hem zo gebruiken

```js
this.messengerState.open
this.messengerState.minimize
```

en kunnen we hem makkelijker uitbreiden/verplaatsen

Reply:
Done 5220f363fe4df2536c8401bbfd8d39e460fa82e1

Reply:
Resolved